### ISSUE TITLE
running 3D-cnn in tensorflow/Keras

### ISSUE THREAD

**Issue by croshong** [gnina/gnina#188]
I have another really basic question

I'm now following the tutorial script with tensorflow as in https://gnina.github.io/libmolgrid/tutorials/train_basic_CNN_with_Tensorflow.html

This script runs without any problem

It seems that with a result of training with each batch (size 50) , loss value was plotted as in example

but when I predict using a model saved in this example, with a data of batch size 50

its predicted value are compose of two probability value for each data point which means for each 50 point, there are two probability 

I think there should be two probability value because in model 

    flatten = keras.layers.Flatten(data_format="channels_first")(conv3)

    fc1 = keras.layers.Dense(2,activation='softmax')(flatten)

as above there should be two output.

But I'm confusing about the meaning of this two value, should I take the max value for each point?

Thanks

**Comment by dkoes** [comment]
I believe the probabilities are the probabilities for the true and false class since they are the result of a softmax.
